joomla/joomla-cms
# [4.0] Cleanup mod_banners

Pull Request for Issue # .

### Summary of Changes
This PR clean up mod_banners after the autoload extension PR merged

1. Namespace ModBannersHelper class
2. Remove the code which register component autoloader

### Testing Instructions
I tested it myself to make sure it is working as expected. So ping @wilsonge to merge it on review to save time of testers
